### DISCUSSION TITLE
battery problem

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#840] [General]
whenever i plug in the battery it locks the screen and disconnects all connections like network bluetooth etc.

**Comment by kRHYME7** [comment]
Can you describe more about your system?

**Comment by sudobrat** [comment]
sure what else do you wanna know?

**Reply by kRHYME7** [reply]
Laptop right? 
How can we reproduce it? 
Did you plug the battery while the system was on?  off?

**Reply by sudobrat** [reply]
Yes asua laptop and I did plug it in while laptop is on

**Reply by kRHYME7** [reply]
Weird, Is that a normal use case?  Or should you plug it in while the laptop is turned off? 
If it's normal I am assuming you're swapping another battery while the system is on right? 

To debug :
try ```  journalctl -f  ```  then plug in/out the battery. With the logs maybe you will know what next step to do.
